package.nls.json

```diff
@@ -32,5 +32,7 @@
     "azureApiManagement.importFunctionApp": "Create from Azure Functions",
     "azureApiManagement.importFunctionAppToApi": "Import Azure Functions",
     "azureApiManagement.addApiToGateway": "Add API to Gateway",
-    "azureApiManagement.removeApiFromGateway": "Remove API from Gateway"
+    "azureApiManagement.removeApiFromGateway": "Remove API from Gateway",
+    "azureApiManagement.importWebApp": "Create from Azure Web Apps",
```

Comment:
Either have everthing as Create or Import

Reply:
WebApp instead of WebApps

Reply:
ok, this naming is for Miao, but we can change this

Reply:
it's not all create or import, create is create new api, import is import to existing ones, which is the same as functions import